### PR TITLE
Test passing

### PR DESCRIPTION
**Pull request by lnchambers**
cbfxryr

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -9,7 +9,7 @@ GEM
       arel (~> 8.0)
     activesupport (5.1.4)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (~> 0.7)
+      i18n (~> 0.9.1)
```
Comment (lnchambers) [inline Gemfile.lock:12]: When I ran bundle, it looks like it updated a few of these

File: app/models/merchant.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class Merchant < ActiveRecord::Base
+  validates_presence_of :name
```
Comment (lnchambers) [inline app/models/merchant.rb:2]: Because we just have the name, this validates it

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20180124043120) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "merchants", force: :cascade do |t|
+    t.text "name"
+  end
```
Comment (lnchambers) [inline db/schema.rb]: Timestamp is missing. Don't merge =/

File: db/schema.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+ActiveRecord::Schema.define(version: 20180124043120) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "merchants", force: :cascade do |t|
+    t.text "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```
Comment (lnchambers) [inline db/schema.rb:22]: There you are, you beautiful, beautiful timestamp bastards

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -9,3 +12,15 @@
 RSpec.configure do |c|
   c.include Capybara::DSL
 end
+
```
Comment (lnchambers) [inline spec/spec_helper.rb:15]: This is to update the test files to include Bundler and DatabaseCleaner

**Review by annaroyer — COMMENTED** [review]
looks great!!!! 💯

**Review by annaroyer — APPROVED** [review]
looks awesomee!! 💯

**Review by garcia50 — COMMENTED** [review]
Good syntax on the test. Lets try and figure out the seeds file next time we meet with Anna.

**Pull request merged**